certified-connectors/IN-D Payables/apiDefinition.swagger.json

```diff
@@ -0,0 +1,365 @@
+{
```

Comment:
@gowthamd3 The swagger is you have uploaded in the certification portal is different than this swagger. Can you please let us know which one is the valid one to consider. If the swagger which was uploaded in the swagger is valid, please replace this swagger.

![image](https://user-images.githubusercontent.com/56276394/107826327-90830080-6d39-11eb-8315-ef38aeb90bb6.png)

Reply:
@gowthamd3 Do you have any update on this?